src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/ReturnApplicationSequenceHandler.cs

```diff
@@ -104,13 +104,39 @@ private async Task NotifyContact(Guid applicationId, int sequenceNo, Cancellatio
                 }
                 else if(sequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
                 {
-                    // TO DO: Send an email to notify organisation withdrawal approval to EPAO
                     var lastSubmission = application.ApplyData?.Apply.LatestOrganisationWithdrawalSubmission;
+                    if (lastSubmission != null)
```

Comment:
We should only be sending the email in the case of this being feedback and not for approval - there is switch above that check for that - similar can be used here

Reply:
Done